### PR TITLE
Feature/team template

### PR DESCRIPTION
**Pull request by TatianaLopaeva**
Added Team Template.

### CODE REVIEW

File: packages/asc-web-common/api/settings/index.js [hunk 1]
```diff
@@ -79,6 +79,49 @@ export function getLogoUrls() {
   });
 }
 
+export function getCustomSchema() {
```
Comment (AlexeySafronov) [inline packages/asc-web-common/api/settings/index.js:82]: Rename to getCustomSchemas -> The method returns a list

File: web/ASC.Web.Client/src/components/pages/Settings/categories/common/customization.js [hunk 2]
```diff
@@ -149,6 +150,8 @@ class Customization extends React.Component {
           this.setState({ languages, language, timezones, timezone });
         });
     }
+    console.log("!!!!", this.state.language);
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/components/pages/Settings/categories/common/customization.js:153]: Remove dev logs

File: web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js [hunk 3]
```diff
@@ -0,0 +1,716 @@
...
+      setCustomShema,
+      teamTemplate,
+      getCurrentCustomSchema,
+      t,
+      setIsLoading,
+    } = this.props;
+
+    if (this.isInvalidForm()) return;
+
+    if (selectedOption.label !== teamTemplate[3].name) {
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js:376]: What is it teamTemplate[3]? Replace magic number

File: web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js [hunk 4]
```diff
@@ -0,0 +1,716 @@
...
+          />
+        </FieldContainer>
+
+        <FieldContainer
+          className="team-template_field-container"
+          isVertical
+          labelText={`${t("Users")}:`}
+          place="top"
+          hasError={formErrors.usersCaption}
+          errorMessage={`${t("EmptyFieldError")}`}
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js:519]: Replace to t("EmptyFieldError")

File: web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js [hunk 5]
```diff
@@ -0,0 +1,716 @@
...
+          />
+        </FieldContainer>
+
+        <FieldContainer
+          className="team-template_field-container"
+          isVertical
+          labelText={`${t("Group")}:`}
+          place="top"
+          hasError={formErrors.groupCaption}
+          errorMessage={`${t("EmptyFieldError")}`}
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js:539]: Replace to errorMessage={t("EmptyFieldError")}

Comment (TatianaLopaeva) [reply web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js]: Replaced.

File: web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js (hunk 4 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js]: Replaced.

File: web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js (hunk 3 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js]: Deleted teamTemplate[3]. Added customName and customId to compare with the currently selected option.

File: web/ASC.Web.Client/src/components/pages/Settings/categories/common/customization.js (hunk 2 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Client/src/components/pages/Settings/categories/common/customization.js]: Deleted.

File: packages/asc-web-common/api/settings/index.js (hunk 1 above)
Comment (TatianaLopaeva) [reply packages/asc-web-common/api/settings/index.js]: Rename to getCustomSchemasList.

**Review by AlexeySafronov — DISMISSED** [review]

**Review by AlexeySafronov — APPROVED** [review]

**Pull request merged**